lib/util/git/index.ts

```diff
@@ -651,23 +659,27 @@ export async function isBranchModified(branchName: string): Promise<boolean> {
     logger.warn({ err }, 'Error checking last author for isBranchModified');
   }
   const { gitAuthorEmail } = config;
-  if (
-    lastAuthor === gitAuthorEmail ||
-    config.ignoredAuthors.some((ignoredAuthor) => lastAuthor === ignoredAuthor)
-  ) {
-    // author matches - branch has not been modified
-    logger.debug('branch.isModified() = false');
-    config.branchIsModified[branchName] = false;
-    setCachedModifiedResult(branchName, false);
-    return false;
-  }
-  logger.debug(
-    { branchName, lastAuthor, gitAuthorEmail },
-    'branch.isModified() = true'
-  );
-  config.branchIsModified[branchName] = true;
-  setCachedModifiedResult(branchName, true);
-  return true;
+  return !!lastAuthors!.find((lastAuthor) => {
```

Comment:
```suggestion
  return !!lastAuthors.find((lastAuthor) => {
```

Reply:
I don't understand what you're trying to achieve with this `find()`. Isn't it "if any authors in this branch aren't expected then branch is modified"? Because the logic below that doesn't seem to match.